LICENSE.txt

```diff
@@ -1,10 +1,10 @@
-   Copyright aio-libs contributors.
+  Copyright 2014-2022 aio-libs contributors.
```

Comment:
The Linux Foundation recommends dropping the year or using just the start date. Updating the year is just busy work.